src/Tribe/Cost_Utils.php

```diff
@@ -346,7 +346,7 @@ public function parse_cost_range( $costs, $max_decimals = null, $sort = true ) {
 		}
 
 		$output_costs = [];
-		$costs        = call_user_func_array( 'array_merge', $costs );
+		$costs        = call_user_func_array( 'array_merge', array_values( $costs ) );
```

Comment:
`array_values()` accepts an array too, so this will likely have a similar issue. It _is_ ok to add a wrapping if statement.

Reply:
I just saw your reply, though. That makes sense as long as `$costs` is an array :aw_yeah: